scripts-available/CDB_Groups_API.sql

```diff
@@ -0,0 +1,57 @@
+-- Requires configuration parameter. Example: SELECT cartodb.CDB_Conf_SetConf('groups_api', '{ "host": "127.0.0.1", "port": 3000, "timeout": 10, "username": "superadmin", "password": "monkey" }');
+
+-- Sends the create group request
+CREATE OR REPLACE
+FUNCTION cartodb._CDB_Group_CreateGroup_API(database_name text, group_name text, group_role text)
+    RETURNS VOID AS
+$$
+    import httplib
+    import base64
+    import string
+    import json
+
+    try:
+      conf = plpy.execute("SELECT cartodb.CDB_Conf_GetConf('groups_api') conf")[0]['conf']
```

Comment:
Well, storing a simple string and recovering it as I want seems simple to me :_) It also allows using different formats in the future. Since parameter usage can be different, I don't see the need for sticking to one. Next one might want to use colon separated parameters for other things, who knows.

Summing up, I feel `text` type is simple and flexible, but I also understand your point. You have a better global vision, I'll use whatever you think is better. @javisantana please choose :-)